kibana-reports/package.json

```diff
@@ -32,6 +32,7 @@
     "@types/dompurify": "^2.0.4",
     "@types/jsdom": "^16.2.4",
     "@types/react-addons-test-utils": "^0.14.25",
+    "@types/uuid": "^8.3.0",
```

Comment:
remove all new uuid related dependencies, you don't need those for `import uuidv1() from uuid/v1`

Reply:
will do